web/themes/joinup/joinup_theme.theme

```diff
@@ -1,6 +1,42 @@
 <?php
-
 /**
  * @file
  * Functions to support theming in the Joinup theme.
  */
+
+use Drupal\image\Entity\ImageStyle;
+use \Drupal\file\Entity\File;
+
+/**
+ * Implements hook_preprocess_page().
+ */
+function joinup_theme_preprocess_page(&$variables) {
+  // Set page title.
+  $request = \Drupal::request();
+  $route_match = \Drupal::routeMatch();
+  $title = \Drupal::service('title_resolver')->getTitle($request, $route_match->getRouteObject());
+  $variables['title'] = $title;
+}
+
+/**
+ * Implements hook_theme_preprocess_menu().
+ */
+function joinup_theme_preprocess_menu(&$variables) {
+  if ($variables['menu_name'] != 'account') {
+    return;
+  }
+  // Set the account logo.
+  $account = \Drupal::currentUser()->getAccount();
+  $user = user_load($account->id(), TRUE);
+  /** @var \Drupal\image\Plugin\Field\FieldType\ImageItem $photo */
+  $photo = $user->get('field_user_photo')->first();
+  $variables['profile_icon'] = '/themes/joinup/images/user-profile-icon.png';
+  if ($photo) {
+    $reference = $photo->getValue();
+    /** @var \Drupal\file\Entity\File $file */
+    $file = File::load($reference['target_id']);
```

Comment:
If the $reference is empty (i.e. the user does not have a picture) then this will throw a warning as the getValue() returns empty. We should either use ->target_id instead of ->getValue() which I think returns the value itself or null or we should check.